src/main/java/com/dropbox/core/stone/StoneDeserializerLogger.java

```diff
@@ -0,0 +1,26 @@
+package com.dropbox.core.stone;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StoneDeserializerLogger {
+    public static Map<Class, LoggerCallback> LOGGER_MAP =
```

Comment:
Can this be private? I know it's just temporary code but would still be super useful.